lms/djangoapps/bulk_email/models.py

```diff
@@ -263,6 +263,9 @@ class Meta(object):
     plain_template = models.TextField(null=True, blank=True)
     name = models.CharField(null=True, max_length=255, unique=True, blank=True)
 
+    def __unicode__(self):
+       return "Template: " + self.name if self.name else "(no name provided)"
```

Comment:
For now I just remove the "Template:" prefix until you decide if you want to change the model.